plugins/modules/remote_management/ipmi/ipmi_power.py

```diff
@@ -51,13 +51,30 @@
             - reset -- Request system reset without waiting for OS
             - boot -- If system is off, then 'on', else 'reset'"
     choices: ['on', 'off', shutdown, reset, boot]
-    required: true
```

Comment:
You need to document both in the description for this option and in the description of the `machine` option that at least one of these two options is required.